### DISCUSSION TITLE
تعیینِِ تکلیفِ موضوعِ خبرنامه

### DISCUSSION THREAD

**Discussion by ghost** [softwaretalks/newsletter#236] [General]
<div dir="rtl">
سلام.

با نگاهی به خبرنامه‌هایِ اخیر می‌شه فهمید موضوعِ اصلیِ اون **برنامه‌نویسی** است و این موضوع چندان ارتباطی با علایقِ گیک‌هایِ کامپیوتر نداره؛ مثلاً «**راهنمای خرید کیبورد مکانیکی برای مبتدیان**» یکی از موضوعاتِ موردعلاقهٔ گیک‌هایِ کامپیوتر هست که تعدادِ مطالبِِ مشابهش در خبرنامه بسیار اندکه. به نظرِ من حوزهٔ کاریِ خبرنامه رو منحصر به برنامه‌نویسی کنید که هدف‌گیریِ جامعهٔ هدف آسون‌تر بشه.
</div>

**Comment by ohmydevops** [comment]
<div dir='rtl'>

سلام جناب کوکبی

من موضوع خاصی رو برای خبرنامه به صورت "تخصصی" متصور نشدم. یعنی قراره خبرنامه Softwaretalks به جنبه‌های مختلف زندگی فنی و غیرفنی مهندسان نرم‌افزار بپردازه. نمیدونم چه کلمه‌ای بهتر میتونه این رو توصیف کنه. من نزدیک‌ترین کلمه برای ذهن رو انتخاب کردم. شاید بهتر باشه کلمه‌ای که موضوع خبرنامه رو توصیف میکنه با دقت و ظرافت بیشتری انتخاب کنیم.

جنبه‌های مختلف زندگی مهندسان‌نرم‌افزار میتونه از مباحث مرتبط با استرس‌کاری به خاطر شرایط این رشته تا نحوه‌ مدیریت پروژه‌های نرم‌افزاری و نقش مهندسان‌ نرم‌افزار در مدیریت تا مباحثی مثل معماری نرم‌افزار رو دربر بگیره. 

اما با کلیت حرف شما کاملا موافق هستم. یک خبرنامه تخصصی میتونه بهتر حوزه تخصصی خودش رو پوشش بده تا خبرنامه‌ای عمومی. اما برای شروع کمی این‌ کار سخت هست به نظرم و باعث میشه خبرنامه‌ به‌وجود نیاد. شاید در طول زمان حل بشه.

همچنین توی پاکت ما میتونیم میزان کلیک‌ها روی لینک‌ها رو ببینیم. شاید بد نباشه یک بررسی آماری از لحاظ موضوعی بکنم و نتیجه‌ رو به صورت عمومی منتشر کنم تا ببینیم سلیقه کاربران به چه صورت بوده بیشتر (البته علت‌های کلیک کردن یا نکردن روی یک لینک میتونه دلایلش خیلی متفاوت‌تر از اهمیت موضوع اون مطلب باشه ولی حداقل یک دید خوبی میده به سلیقه و علایق کاربران)

نظر شما چیه؟‌ @Hameds 
</div>

**Reply by Hameds** [reply]
باید هفکری داشته باشیم. الان نظر خاصی ندارم
